java-client/flight-examples/build.gradle

```diff
@@ -51,13 +50,9 @@ applicationDistribution.into('bin') {
     from(createApplication('add-to-input-table', 'io.deephaven.client.examples.AddToInputTable'))
     from(createApplication('kv-input-table', 'io.deephaven.client.examples.KeyValueInputTable'))
 
-    from(createApplication('get-ticket-table', 'io.deephaven.client.examples.GetTicketTable'))
+    from(createApplication('get-table', 'io.deephaven.client.examples.GetDirectTable'))
+    from(createApplication('get-schema', 'io.deephaven.client.examples.GetDirectSchema'))
```

Comment:
next chance you get, consider generalizing the CreateStartScripts so these mains run cleanly from IJ clicking on the green triangle next to the name - should be a matter of assigning the global application fields like classpath, jvm args, etc as the jetty/netty modules do, and have the scripts reference those rather than have each task have its own copy and none be global

Reply:
So, these are runnable running from IJ (maybe they don't inherit the same default jvm args as we specify in gradle, I'll grant that).

Clicking the green run for JettyMain - that doesn't inherit applicationDefaultJvmArgs, does it?

Reply:
...you are right, that's irritating, I had understood that to use the application plugin configuration to exec, but instead it is some synthetic task, `':server-jetty:JettyMain.main()'`, which isn't in the actual gradle model (but IJ makes it work anyway...).

That task is apparently a JavaExec, so the java-open-nio applies, and the same trick can be used for consistent jvm args (like forcing groovy):

```
tasks.withType(JavaExec) {
    jvmArgs applicationDefaultJvmArgs
}
```
(this is in server/jetty/build.gradle, and just grabs the application args, etc).